client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/styles.scss

```diff
@@ -163,14 +163,24 @@
 			text-decoration: underline;
 		}
 
-
-		.components-button:has(.delete-label),
-		.components-button:has(.refresh-label),
-		.components-button:has(.unlock-label), {
-			color: var(--gray-gray-100, #101517);
-			text-decoration-color: var(--gray-gray-100, #101517);
+		.validation-actions {
+			margin-left: 5px;
+			@media (max-width: $break-medium ) {
+				display: block;
+				margin-left: 0;
+			}
+			.components-button:has(.delete-label),
+			.components-button:has(.refresh-label),
+			.components-button:has(.unlock-label), {
+				color: var(--gray-gray-100, #101517);
```

Comment:
Color looks like a typo? it's `color: var(--studio-gray-100);`.

Reply:
Thank you!